tests/e2e/testutils.go

```diff
@@ -0,0 +1,208 @@
+package e2e
```

Comment:
Is this package / file necessary since everything is exported, there's no API really.

Reply:
Future tests need to be able to pull it in. I'm not quite sure what your proposing. Yes everything is exported, that's the point.

Reply:
Generally speaking, packages in Go are meant to define API boundaries. When everything is exported, it negates the need for import as the code can live next to where it's used. If it's going to be imported into other tests, then that makes sense, assuming the other tests will be in a different package from the existing tests that use this code. If they don't, this can live along side the test code that uses it.

Reply:
Other tests will be in other packages/folders. It will be how the e2e tests in K3s are https://github.com/k3s-io/k3s/tree/master/tests/e2e. This is a limitation of Ginkgo assuming only one type of test can exist within a package. Vagrant also suffers from this limitation with "one" Vagrantfile per folder.

Reply:
Gotcha. Makes sense then. I just want to avoid the creation of packages for file management purposes rather than creating APIs which they're meant for.